config/software/ncurses.rb

```diff
@@ -24,6 +24,9 @@
 
 dependency "config_guess"
 
+# versions_list: https://ftp.gnu.org/gnu/ncurses/ filter=*.tar.gz
+version("6.2") { source sha256: "30306e0c76e0f9f1f0de987cf1c82a5c21e1ce6568b9227f7da5b71cbea86c9d" }
```

Comment:
So if you notice for the existing versions, the URL is included in the version block. We'll need to make sure to we either a) include a default source url (like we do for other definitions) or b) include the url in the block. 

I would defer to (a):

```
source url: "https://ftp.gnu.org/gnu/ncurses/ncurses-#{version}.tar.gz"
```